### PR TITLE
added serveo server (ssh tunneling)

### PR DESCRIPTION
**Pull request by usama7628674**
Now socialfish has 2 servers ngrok & serveo.

### CODE REVIEW

File: SocialFish.py [hunk 1]
```diff
@@ -284,16 +284,38 @@ def runNgrok():
     url = open('ngrok.url', 'r')
     print("\n {0}[{1}*{0}]{1} Ngrok URL: {2}".format(CYAN, END, GREEN) + url.read() + "{1}".format(CYAN, END, GREEN))
     url.close()
+    
+def serveo():
+    system('ssh -R 80:localhost:1111 serveo.net > sendlink.txt 2> /dev/null & ')
+    sleep(4)
```
Comment (alexmozzhakov) [inline SocialFish.py]: is it safe?

**Comment by alexmozzhakov** [comment]
Idk why we need ngrok at all then. According to http://serveo.net/ "The primary advantage of Serveo over ngrok is the use of your existing SSH client, so there's no client application to install.", so we don't need to mess with termux version then, I think.

File: SocialFish.py [hunk 2]
```diff
@@ -284,16 +284,38 @@ def runNgrok():
...
+    f = open('sendlink.txt', 'r')
+    a = ['[32m', 'Forwarding', 'HTTP', 'traffic', 'from', '[0m', ' ']
+    lst = []
+    for line in f:
+    	for word in a:
+            if word in line:
+            	line = line.replace(word,'')	    	
+    	lst.append(line)
+    f.close()
+    f = open('sendlink.txt','w')
```
Comment (alexmozzhakov) [inline SocialFish.py]: why do you have to have this file?

Comment (usama7628674) [reply SocialFish.py]: Also we should remove ngrok and shift from php to lighttpd server and then we get significant speed boost and pages load quickly.There is delay in termux version of ngrok so after this PR I'll work on next PR and change server to lighttpd.

File: SocialFish.py (hunk 1 above)
Comment (usama7628674) [reply SocialFish.py]: Safe in what way?

File: SocialFish.py (hunk 2 above)
Comment (usama7628674) [reply SocialFish.py]: What do you mean by that?
so we don't need to mess with termux version then, I think

File: SocialFish.py (hunk 1 above)
Comment (alexmozzhakov) [reply SocialFish.py]: we sleep for 4 seconds waiting for some connection to happen

Comment (usama7628674) [reply SocialFish.py]: Yes for ssh to save that link in sendfile.txt

Comment (usama7628674) [reply SocialFish.py]: and to generate link will take 2-3 seconds

File: SocialFish.py (hunk 2 above)
Comment (alexmozzhakov) [reply SocialFish.py]: @usama7628674 I mean if we delete ngrok there won't be any differences between termux and regular version. The script doesn't open any files for write in `runNgrok()` after pipe and there is no point in doing so.

Comment (usama7628674) [reply SocialFish.py]: @alexmozzhakov Yeah I didn't notice that one.We just need to have one .py file for linux and termux

Comment (usama7628674) [reply SocialFish.py]: I just pushed another PR to remove ngrok

File: SocialFish.py (hunk 1 above)
Comment (usama7628674) [reply SocialFish.py]: For ngrok we sleep for 10 seconds now 4 seconds.It's hardly noticeable.

Comment (alexmozzhakov) [reply SocialFish.py]: for i = 0; i < 100; i+=1 {
  if exists(file) break;
  sleep(100)
}
if !exists(file) timeout();

this is the pseudo code for what I'm asking

Comment (alexmozzhakov) [reply SocialFish.py]: As far as I remember we cannot assume internet connection for a user at this point, so it would be much user-friendly.

Comment (usama7628674) [reply SocialFish.py]: OK I merge this PR and you make another PR for this change and I will test it and merge it

Comment (alexmozzhakov) [reply SocialFish.py]: What? Why?

Comment (usama7628674) [reply SocialFish.py]: I deleted socialfishtermux.py cuz form now on only one file is compatible for termux and linux.

Comment (alexmozzhakov) [reply SocialFish.py]: @usama7628674 Not util you merge both PR's, and removal should go into second one, not right away

Comment (alexmozzhakov) [reply SocialFish.py]: @usama7628674 And this one isn't ready IMO, we still have useless file write for sure.

Comment (usama7628674) [reply SocialFish.py]: ngrok.url is the one

Comment (alexmozzhakov) [reply SocialFish.py]: No, ngrok.url is piped, you don't write there, with `open('sendlink.txt','w')`

Comment (usama7628674) [reply SocialFish.py]: yes just checked all the files.

**Pull request merged**